lib/internal/bootstrap_node.js

```diff
@@ -54,6 +54,8 @@
 
     _process.setupRawDebug();
 
+    NativeModule.require('internal/url');
```

Comment:
There shouldn't be. The requires that are there are minimal (`util` will have already been loaded, `os` is small, etc). I'm not convinced this should be a concern.